temporalcli/commands.go

```diff
@@ -576,3 +576,16 @@ func encodeMapToPayloads(in map[string]any) (map[string]*commonpb.Payload, error
 	}
 	return out, nil
 }
+
+type overrideDisplayTypeFlagValue struct {
+	pflag.Value
+	displayType string
+}
+
+func (o *overrideDisplayTypeFlagValue) Type() string {
+	return o.displayType
+}
+
+func OverrideFlagDisplayType(flag *pflag.Flag, displayType string) {
```

Comment:
Realized this is same package, could make this non-exported (i.e. lowercase), but not a big deal